.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Description
+
+Please provide a brief description of the changes in this pull request.
+
+### Motivation
+
+Explain the motivation behind these changes and why they are necessary.
+
+### Type of Change
+
+- [ ] Bug fix (non-breaking change that resolves an issue)
+- [ ] Feature enhancement (non-breaking change that adds functionality)
+- [ ] Documentation update (non-breaking change that improves documentation)
+- [ ] Refactoring (non-breaking change that improves code structure)
+- [ ] Other (please specify)
+
+### Checklist
+
+Please review the following checklist and ensure that all items are completed before submitting the pull request:
+
+- [ ] My changes do not introduce any breaking changes or change existing API behaviors.
+    - Keep in mind, There are rather large users of lol-html so even small changes to API behavior can be breaking
```

Comment:
> rather large users

this sounds quite arrogant, let's just replace with something like "big userbase"